plugin.py

```diff
@@ -74,7 +74,7 @@ def _to_lsp_edits(self, items: Any) -> Dict[str, List[TextEditTuple]]:
                         (
                             (start['line'] - 1, start['offset'] - 1),
                             (end['line'] - 1, end['offset'] - 1),
-                            text_change['newText'],
+                            text_change['newText'].replace("\r", ""), # remove \r to prevent Windows from inserting `<0x0d>`
```

Comment:
The comment is very redundant.

Reply:
If you want to have a comment then having the same one as here would be preferred: https://github.com/sublimelsp/LSP/blob/53b3499b0142c179f5d854f8fd8594ed5c67b17b/plugin/core/edit.py#L44-L45

Reply:
I will just remove the comment